src/jit/codegenxarch.cpp

```diff
@@ -1437,8 +1437,7 @@ CodeGen::isStructReturn(GenTreePtr treeNode)
     }
```

Comment:
Comment on lines 1422-1424 needs modification, since this method no longer returns true for methods that return RetBufArg addr.